### ISSUE TITLE
Authenticator binding description requirements

### ISSUE THREAD

**Issue by dig-comments** [usnistgov/800-63-3#804]
**Organization Name (N/A, if individual)**: National Security Agency

**Organization Type**: 1
 
**Document (63-3, 63A, 63B, or 63C)**: 800-63B

**Reference (Include section and paragraph number)**: 6.1, 2
 
**Comment (Include rationale for comment)**:  Suggest rewording slightly to read:
 
 "The record for each authenticator created by the CSP SHALL contain the date and time the authenticator was bound to the account/identity. The record SHOULD include information about the binding, such as the IP address and device identifier of any device associated with the enrollment. If available, the record SHOULD also contain information about unsuccessful authentications attempted with the authenticator." The last sentence in this paragraph seems to contradicts the last sentence in the previous paragraph which states "The CSP or verifier SHALL also maintain the information required for throttling authentication attempts when required, as described in section 5.2.2." SHALL has gone to SHOULD in the original text below.

Original text:
"The record created by the CSP SHALL contain the date and time the authenticator was bound to the account and SHOULD include information about the binding, such as the IP address and any device identifier associated with the enrollment. If available, the record SHOULD also contain information about unsuccessful authentications attempted with the authenticator.""

**Suggested Change**: Make suggested change.
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

**Comment by jimfenton** [comment]
With respect to the apparent contradiction, what was intended was that the required information to support throttling would include the number and timing of unsuccessful attempts, while the recommended information would include information about the IP address and device involved. The wording is being changed to make that clearer.
